linking/static/grpc/linked-against-grpc.yml

```diff
@@ -0,0 +1,27 @@
+rule:
+  meta:
+    name: linked against gRPC
+    namespace: linking/static/grpc
+    authors:
+      - jakubjozwiak@google.com
+    scopes:
+      static: file
+      dynamic: file
+    references:
+      - https://grpc.io
+      - https://github.com/grpc/grpc
+    examples:
+      - 71419e4d6f538b8a04ca6786e4d68f0d40204df465fe80bc6b4a0b259103e50f
+  features:
+    - or:
```

Comment:
```suggestion

```

Reply:
Thank you! Updated in https://github.com/mandiant/capa-rules/pull/1068/commits/eb07af0272400b1ab47e2926be71e946fac3f526